### ISSUE TITLE
Google Auth - Instructions to create "Service account" can be provided

### ISSUE THREAD

**Issue by sowmyav27** [rancher/rancher#21389]
**What kind of request is this (question/bug/enhancement/feature request):** bug

**Steps to reproduce (least amount of steps as possible):**

1. Login to Rancher UI
2. Go to Global --> Authentication --> Google
4. steprancher/rancher#4                      - `Navigate to the "Credentials" tab again to create your Service account key:`

Not enough instructions are available for the user to create a Service account key. 

More details or a Link to google docs, as to how to create a Service account can be provided for the users.
<img width="1306" alt="Screen Shot 2019-07-09 at 7 48 03 AM" src="https://user-images.githubusercontent.com/26032343/60898450-e9b1b680-a21d-11e9-9c52-2b7ef786467d.png">

**Other details that may be helpful:**

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): master-head
- Installation option (single install/HA): single

**Comment by sowmyav27** [comment]
Reopening the issue, as a more specific link to the page - to create a service account and the 5 scopes to set up the service account should be available in the instructions for the user.

**Comment by loganhz** [comment]
@sowmyav27 Cloud you provide more info about what you want for this issue, please? It will be good if you can provide the exact text content.

**Comment by sowmyav27** [comment]
@loganhz Instructions to setup the service account has been elaborated by @mrajashree here - https://gist.github.com/mrajashree/9dc4cd8e11a9e813a96e713964c74ca9. A similar set of instructions on the Rancher UI would be helpful for any user to set up a service account.

**Comment by loganhz** [comment]
@sowmyav27 

Do you want to add something in Point 4?
Or would you like to create a new Point 5?

If you would like to create Point 5, could you provide the detailed text for :

- The main row
- Sub rows


Such as 

```
Point 4: Navigate to the "Credentials" tab again to create your Service account key:

- Click here, for information on creating and managing your service account.
- Select your service account.
- Select JSON for your Key type and then click "Create".
- A JSON file will be automatically saved locally.
```

**Comment by davidnuzik** [comment]
I think we were also considering linking to rancher docs as well as an option (since there are so many things to do on this last step). This is another way we could do it so we don't have to clutter the rancher UI too much

**Comment by loganhz** [comment]
@davidnuzik could you provide the detail design and text for this, please? Thanks!

**Comment by davidnuzik** [comment]
I will try; I am not a good person to ask this. I do not really know what all I have to do to set up it will take some time.

**Comment by davidnuzik** [comment]
We are going to review internally and provide feedback soon.

**Comment by mrajashree** [comment]
I think having a link to docs might look better. That's what we do for instructions on service account creation for GKE clusters

**Comment by tfiduccia** [comment]
@loganhz - lauren will add wireframe.  The link to the docs will be "https://rancher.com/docs/rancher/v2.x/en/admin-settings/authentication/google/"

**Comment by lvuch** [comment]
https://xd.adobe.com/view/1036add8-5175-4b0e-620b-53f139c74cd9-6607/

**Comment by davidnuzik** [comment]
Could we possibly decrease the amount of white space (vertical space) between each step? It seems like somewhat large of an amount of white space.

![Screenshot_2019-08-29_15-11-27](https://user-images.githubusercontent.com/45179589/63980263-e7c2e180-ca6f-11e9-863b-8e0bcfab9594.png)

Also, will we not be changing to this style in all areas (all auth providers?) why is just this one going to be different?
cc: @tfiduccia

**Comment by tfiduccia** [comment]
@davidnuzik - All providers will be changing.  Just not for 2.3, that will be future work.
@lvuch - White space?

**Comment by lvuch** [comment]
@tfiduccia @davidnuzik See the above PR. Reduce white space by half and adjusted tooltips as requested by @tfiduccia

**Comment by davidnuzik** [comment]
Thanks! If there's anything else we'll open a separate issue if necessary.
(Waiting on PR to be merged currently)

**Comment by davidnuzik** [comment]
Version: master-head (v2.3) (9/3/19)

This changes the layout of the Googleoauth page. It makes things a little simpler and now we just have three steps outlined.
There is also a new banner on top too that will refer to docs we will make available before release of the product. This should help to make the process smoother as well.

Here is a screenshot showing the changes. Basically post v2.3 release we'll start going thru the other providers and making them look similar to how Google looks here.

![Screenshot_2019-09-03_15-24-51](https://user-images.githubusercontent.com/45179589/64212803-5a143700-ce5f-11e9-9b75-888de1aba206.png)

All links work as expected. I didn't identify any issues.
